packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1369,7 +1376,10 @@ export function useGlobalStylesOutputWithConfig(
 			hasBlockGapSupport,
 			hasFallbackGapSupport,
 			disableLayoutStyles,
-			disableRootPadding
+			options?.disableRootPadding,
+			{
```

Comment:
Note to self: I think `toStyles` is also ripe for a small refactor. Moving the following arguments

```
	hasBlockGapSupport,
	hasFallbackGapSupport,
	disableLayoutStyles = false,
	disableRootPadding = false,
```

into `styleOptions`